Volume/View Models/ArticleRow.swift

```diff
@@ -11,15 +11,17 @@ import SwiftUI
 
 struct ArticleRow: View {
     let article: Article
+    let entryPoint: EntryPoint
```

Comment:
I like `NavigationSource` just a little more, but that's totally subjective and I like this name too.